pfrl/agents/hiro_high_level_goal_conditioned_td3.py

```diff
@@ -184,6 +184,11 @@ def high_level_update_q_func_with_goal(self, batch):
         self.q_func1_loss_record.append(float(loss1))
         self.q_func2_loss_record.append(float(loss2))
 
+        q1_recent_variance = np.var(list(self.q1_record)[-100:])
+        q2_recent_variance = np.var(list(self.q2_record)[-100:])
```

Comment:
why 100? Give it a name for this constant.

Reply:
fixed